lms/templates/learner_dashboard/programs.html

```diff
@@ -20,6 +20,16 @@
 <%block name="pagetitle">${_("Programs")}</%block>
 
 <div class="program-list-wrapper">
+    % if not len(programs):
+        <div class="dashboard">
+        <section class="empty-dashboard-message">
+            <p>${_("It looks like you aren't yet enrolled in any XSeries Programs!")}</p>
+              <a class="find-xseries-programs" href=""><i class="action-xseries-icon" aria-hidden="true"></i>
```

Comment:
@jimabramson Thanks, I think this is what I was looking for https://github.com/edx/edx-platform/blob/393e885e4b6e701968150db09b87c93d227edb95/lms/djangoapps/learner_dashboard/views.py#L35